code/aind_smartspim_quantification/utils/utils.py

```diff
@@ -447,19 +471,20 @@ def get_mesh_interrior_points(mesh) -> tuple:
     """
 
     bounds = mesh.bounds()
-    region_array = mesh.binarize(spacing = (1,1,1)).tonumpy()
-    
+    region_array = mesh.binarize(spacing=(1, 1, 1)).tonumpy()
+
     indecies = np.where(region_array == 255)
     xs = indecies[0] + int(bounds[0])
     ys = indecies[1] + int(bounds[2])
     zs = indecies[2] + int(bounds[4])
-    
+
     return (xs, ys, zs)
 
-def get_intensity_mask(vertices, faces, mask, split) -> np.array:
```

Comment:
I think at this point it is important to think about image utilities and utilities since it gets confusing. You can work on this later not on this PR. Are we planning to add more metrics?

Reply:
That is a good idea, and will work on for a new PR. Not at the moment, but I was thinking it might be good to output some png's for mutual information  overlays similar to what you do in registration which can go into image utils as well. 

I don't think we have any more metrics planned at the moment but I would say the more the better